graphql_compiler/tests/test_helpers.py

```diff
@@ -281,6 +283,174 @@ def get_schema():
+            sqlalchemy.Column('name', sqlalchemy.String(40), nullable=False),
+            schema='db_1.schema_1'
+        ),
+        'Entity': sqlalchemy.Table(
+            'Entity',
+            sqlalchemy_metadata,
+            sqlalchemy.Column('uuid', sqlalchemy.String(36), primary_key=True),
+            sqlalchemy.Column('name', sqlalchemy.String(40), nullable=False),
+            sqlalchemy.Column('related_entity', sqlalchemy.String(36), nullable=True),
+            sqlalchemy.Column('__source_table_name', sqlalchemy.String(36), nullable=False),
+            schema='db_1.schema_1'
+        ),
+        'Event': sqlalchemy.Table(
+            'Event',
+            sqlalchemy_metadata,
+            sqlalchemy_metadata,
+            sqlalchemy.Column('uuid', sqlalchemy.String(36), primary_key=True),
+            sqlalchemy.Column('event_date', sqlalchemy.DateTime, nullable=False),
+            schema='db_1.schema_1'
+        ),
+        'FeedingEvent': sqlalchemy.Table(
+            'FeedingEvent',
+            sqlalchemy_metadata,
+            sqlalchemy.Column('uuid', sqlalchemy.String(36), primary_key=True),
+            sqlalchemy.Column('name', sqlalchemy.String(40), nullable=False),
+            sqlalchemy.Column('event_date', sqlalchemy.DateTime, nullable=False),
+            schema='db_1.schema_1'
+        ),
+        'Food': sqlalchemy.Table(
+            'Food',
+            sqlalchemy_metadata,
+            sqlalchemy.Column('uuid', sqlalchemy.String(36), primary_key=True),
+            sqlalchemy.Column('name', sqlalchemy.String(40), nullable=False),
+            schema='db_1.schema_1'
+        ),
+        'FoodOrSpecies': sqlalchemy.Table(
+            'FoodOrSpecies',
+            sqlalchemy_metadata,
+            sqlalchemy.Column('uuid', sqlalchemy.String(36), primary_key=True),
+            sqlalchemy.Column('name', sqlalchemy.String(40), nullable=False),
+            schema='db_1.schema_1'
+        ),
+        'Location': sqlalchemy.Table(
+            'Location',
+            sqlalchemy_metadata,
+            sqlalchemy.Column('uuid', sqlalchemy.String(36), primary_key=True),
+            sqlalchemy.Column('name', sqlalchemy.String(40), nullable=False),
+            schema='db_1.schema_1'
+        ),
+        'Species': sqlalchemy.Table(
+            'Species',
+            sqlalchemy_metadata,
+            sqlalchemy.Column('uuid', sqlalchemy.String(36), primary_key=True),
+            sqlalchemy.Column('name', sqlalchemy.String(40), nullable=False),
+            sqlalchemy.Column('eats', sqlalchemy.String(36), nullable=True),
+            schema='db_1.schema_1'
+        ),
+        'UniquelyIdentifiable': sqlalchemy.Table(
+            'UniquelyIdentifiable',
+            sqlalchemy_metadata,
+            sqlalchemy.Column('uuid', sqlalchemy.String(36), primary_key=True),
+            schema='db_1.schema_1'
+        ),
+        'Union__BirthEvent__Event__FeedingEvent': sqlalchemy.Table(
```

Comment:
Why do you need to map union types to SQLAlchemy tables? I understand that you need union types in order to be able to have tests for sql that are almost identical to the orientdb tests. However, I don't understand why you chose to have a concrete backend implementation of union types.

Reply:
Good question. I think it was just a convenience, so I can reuse the same tests. I will remove them for now and try to make the tests work without them when I get to that.